### ISSUE TITLE
No pyright commands available even though pyright is working

### ISSUE THREAD

**Issue by gusostow** [fannheyward/coc-pyright#834]
I just installed pyright for the first time and it seems to be working, except none of its commands are available in `:CocCommand`. Am I doing something wrong?

**What's the output of `:CocCommand pyright.version`**
```
[coc.nvim] Command: pyright.version not found
```

**What's the output of `:CocCommand workspace.showOutput Pyright`**
```
Workspace: /home/aostow/dev/lib
Using python from PREFIX/bin/python

[Info  - 4:47:04 PM] Pyright language server 1.1.275 starting
[Info  - 4:47:04 PM] Server root directory: /home/aostow/.config/coc/extensions/node_modules/coc-pyright/node_modules/pyright/dist/
[Info  - 4:47:04 PM] Starting service instance "lib"
[Info  - 4:47:04 PM] No configuration file found.
[Info  - 4:47:04 PM] No pyproject.toml file found.
[Info  - 4:47:04 PM] Setting pythonPath for service "lib": "PREFIX/bin/python"
[Warn  - 4:47:04 PM] stubPath /home/aostow/dev/lib/typings is not a valid directory.
[Info  - 4:47:04 PM] Assuming Python version 3.10
[Info  - 4:47:04 PM] Assuming Python platform Linux
[Info  - 4:47:04 PM] Searching for source files
[Info  - 4:47:04 PM] Found 1371 source files
[Info  - 8:27:01 PM] Starting service instance "config"
[Info  - 8:27:01 PM] No configuration file found.
[Info  - 8:27:01 PM] No pyproject.toml file found.
[Info  - 8:27:01 PM] Setting pythonPath for service "config": "PREFIX/bin/python"
[Warn  - 8:27:01 PM] stubPath /home/aostow/dev/config/typings is not a valid directory.
[Info  - 8:27:01 PM] Assuming Python version 3.10
[Info  - 8:27:01 PM] Assuming Python platform Linux
[Info  - 8:27:01 PM] Searching for source files
[Info  - 8:27:01 PM] Found 50 source files

```

**Comment by fannheyward** [comment]
What's your coc-pyright version? You can get it from `:CocList extensions`.

Run `:CocUpdate` to upgrade coc-pyright.

**Comment by gusostow** [comment]
Thanks that fixed it.

**Comment by ranelpadon** [comment]
FYI, encountered similar issue, even though I'm using the latest version (`1.1.284`), and the `:CocCommand` has no entries for `pyright`, even `:CocCommand workspace.showOutput Pyright` outputs nothing.

Only found out the root cause when I ran `:CocInfo`/`CocOpenLog`:
```js
2023-01-22T03:37:21.969 ERROR (pid:84989) [extensions] - Error on createExtension coc-pyright 
from /Users/ranelpadon/.config/coc/extensions/node_modules/coc-pyright/lib/index.js:11715
          charset = options ?? "utf-8";
                             ^
SyntaxError: Unexpected token '?'
```

Turned out that the `??` operator was introduced in `node v14` and I'm using `node v13` which we need for our project. Also, found out that the `??` was added only in the `coc-pyright v1.1.283`, so I installed the `v1.1.282`, and it worked!
```vim
:CocInstall coc-pyright@1.1.282
```

Maybe would help to indicate also the required `node` version in the **README**? 

PS: thanks for this powerful plugin!
